cobra/cmd/license_lgpl.go

```diff
@@ -0,0 +1,188 @@
+package cmd
+
+func initLgpl() {
+	Licenses["lgpl"] = License{
+		Name:            "GNU Lesser General Public License",
+		PossibleMatches: []string{"lgpl", "lesser gpl", "gnu lgpl"},
```

Comment:
And for plain LGPL in GNU you can see LGPL 3.0: https://www.gnu.org/licenses/lgpl.txt